.github/workflows/push.yaml

```diff
@@ -5,6 +5,11 @@ on: # yamllint disable-line rule:truthy
     branches: [main]
   pull_request_target:
     types: [opened, synchronize, reopened, ready_for_review]
+  workflow_dispatch:
+    inputs:
+      version:
```

Comment:
should we validate this input?

Reply:
The only thing that makes sense to me is ensuring that version inputs are semver-compatible and aren't already in a release.  I'll look into adding it, but it might make sense to punt this to a separate PR so we can get a release out.

Reply:
I agree, let's do that in a separate PR